components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -222,7 +221,7 @@ lnd/clm2/surfdata_map/surfdata_4x5_simyr2000_c130927.nc</fsurdat>
 lnd/clm2/surfdata_map/surfdata_10x15_simyr2000_c130927.nc</fsurdat>
 
 <fsurdat hgrid="ne30np4"     sim_year="2000" >
-lnd/clm2/surfdata_map/surfdata_ne30np4_simyr2000_c180306.nc</fsurdat>
```

Comment:
I had to roll back this surface data file because the new file produces errors like:

```
surfrd_get_topo ERROR: topo file mismatch lat,lon   12.8291796067501
  12.8291796067501        360.000000000000       0.000000000000000E+000
 1.000000000000000E-012
ENDRUN:
ERROR in surfrdMod.F90 at line 513
```

Due to variables needed by the glacier multiple elevation classes (GLC_MEC; used for downscaling), which you can see in this diff: 

```diff
diff --git a/17.head b/18.head
index 7a50cb1..4838843 100644
--- a/17.head
+++ b/18.head
@@ -1,8 +1,6 @@
-netcdf surfdata_ne30np4_simyr2000_c171002 {
+netcdf surfdata_ne30np4_simyr2000_c180306 {
 dimensions:
 	gridcell = 48602 ;
-	nglcec = 10 ;
-	nglcecp1 = 11 ;
 	numurbl = 3 ;
 	nlevurb = 5 ;
 	numrad = 2 ;
@@ -228,27 +226,6 @@ variables:
 	double PCT_GLACIER(gridcell) ;
 		PCT_GLACIER:long_name = "percent glacier" ;
 		PCT_GLACIER:units = "unitless" ;
-	double GLC_MEC(nglcecp1) ;
-		GLC_MEC:long_name = "Glacier elevation class" ;
-		GLC_MEC:units = "m" ;
-	double PCT_GLC_MEC(nglcec, gridcell) ;
-		PCT_GLC_MEC:long_name = "percent glacier for each glacier elevation class (% of landunit)" ;
-		PCT_GLC_MEC:units = "unitless" ;
-	double PCT_GLC_MEC_GIC(nglcec, gridcell) ;
-		PCT_GLC_MEC_GIC:long_name = "percent smaller glaciers and ice caps for each glacier elevation class (% of landunit)" ;
-		PCT_GLC_MEC_GIC:units = "unitless" ;
-	double PCT_GLC_MEC_ICESHEET(nglcec, gridcell) ;
-		PCT_GLC_MEC_ICESHEET:long_name = "percent ice sheet for each glacier elevation class (% of landunit)" ;
-		PCT_GLC_MEC_ICESHEET:units = "unitless" ;
-	double PCT_GLC_GIC(gridcell) ;
-		PCT_GLC_GIC:long_name = "percent ice caps/glaciers (% of landunit)" ;
-		PCT_GLC_GIC:units = "unitless" ;
-	double PCT_GLC_ICESHEET(gridcell) ;
-		PCT_GLC_ICESHEET:long_name = "percent ice sheet (% of landunit)" ;
-		PCT_GLC_ICESHEET:units = "unitless" ;
-	double TOPO_GLC_MEC(nglcec, gridcell) ;
-		TOPO_GLC_MEC:long_name = "mean elevation on glacier elevation classes" ;
-		TOPO_GLC_MEC:units = "m" ;
 	double TOPO(gridcell) ;
 		TOPO:long_name = "mean elevation on land" ;
 		TOPO:units = "m" ;
@@ -273,15 +250,15 @@ variables:

 // global attributes:
 		:Conventions = "NCAR-CSM" ;
-		:History_Log = "created on: 10-02-17 15:02:49" ;
-		:Logname = "gb9" ;
-		:Host = "titan-batch2" ;
+		:History_Log = "created on: 03-06-18 08:34:49" ;
+		:Logname = "shix" ;
+		:Host = "edison04" ;
 		:Source = "Community Land Model: CLM4" ;
 		:Version = "$HeadURL: https://svn-ccsm-models.cgd.ucar.edu/clm2/trunk_tags/clm4_5_1_r085/models/lnd/clm/tools/clm4_5/mksurfdata_map/src/mkfileMod.F90 $" ;
 		:Revision_Id = "$Id: mkfileMod.F90 47951 2013-06-12 11:13:58Z sacks $" ;
 		:Compiler_Optimized = "TRUE" ;
 		:no_inlandwet = "TRUE" ;
-		:nglcec = 10 ;
+		:nglcec = 0 ;
 		:Input_grid_dataset = "map_0.5x0.5_landuse_to_ne30np4_aave_da_110320.nc" ;
 		:Input_gridtype = "global" ;
 		:VOC_EF_raw_data_file_name = "mksrf_vocef_0.5x0.5_simyr2000.c110531.nc" ;
@@ -324,5 +301,5 @@ variables:
 		:Soil_order_raw_data_file_name = "mksrf_soilord_global_c150313.nc" ;
 		:Fmax_raw_data_file_name = "mksrf_fmax_3x3min_USGS_c120911.nc" ;
 		:Organic_matter_raw_data_file_name = "mksrf_organic_10level_5x5min_ISRIC-WISE-NCSCD_nlev7_c120830.nc" ;
-		:Vegetation_type_raw_data_filename = "mksrf_landuse_rc2000_c090630.nc" ;
+		:Vegetation_type_raw_data_filename = "AA_mksrf_landuse_rc_2000_06062017.nc" ;
 }

```

@acme-y9s , did you make this new `fsurdat` file? I think `shix` is your NESC username

Reply:
Looks like the regenerated `fsurdat` file works well! Thanks @bishtgautam